docs/tutorials/index.rst

```diff
@@ -6,8 +6,9 @@ Tutorials
 .. toctree::
    :maxdepth: 1
 
-   get-started
+   install
    first-steps
+   synchronisation
```

Comment:
Yes, I did that in https://github.com/Kinto/kinto/pull/585/commits/894f7c18df3abe636bca40bfc0349ec505a2c48a.